index.html

```diff
@@ -414,7 +438,10 @@ <h3>MathML Element Mappings</h3>
 		  </td>
 		  <td class="axapi">
 		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
-		    <span class="subrole">AXSubrole: <code>AXMathSquareRoot</code></span>
+		    <span class="subrole">AXSubrole: <code>AXMathSquareRoot</code></span><br />
+		    <span class="attrs">AXAttributes:
+		      <code>AXMathRootRadicand</code> pointing to a first child
```

Comment:
MathML allows `<msqrt>child0 child1 child2 ... childN</msqrt>`. An anonymous node is *not* created to wrap the children. Not sure how that translates in the a11y tree and what NSAccessibility actually expects...